clients/hadoopfs/pom.xml

```diff
@@ -222,23 +240,36 @@ To export to S3:
             <artifactId>api-client</artifactId>
             <version>0.81.0</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-lang3</artifactId>
+            <version>3.12.0</version>
+        </dependency>
 
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
-            <version>2.7.7</version> <!-- The common Hadoop version for Spark 2.4.7 and 3.0.1, which lakeFS supports -->
+            <version>${hadoop.version}</version>
+            <scope>provided</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.hadoop</groupId>
             <artifactId>hadoop-common</artifactId>
             <scope>test</scope>
             <type>test-jar</type>
-            <version>2.7.7</version>
+            <version>${hadoop.version}</version>
         </dependency>
```

Comment:
I think we can keep only the `provided` one as it can also be used for tests
https://maven.apache.org/guides/introduction/introduction-to-dependency-mechanism.html#Dependency_Scope

Reply:
This one is needed for the "test-jar" type.